---
title: Join
nav:
  order: 6
  tooltip: Open positions
---

# {% include icon.html icon="fa-solid fa-handshake-angle" %}Join the Team

We are always seeking talented and passionate people to join our team, from summer students to postdocs to staff scientists, short-term and long-term. You can learn a bit about working in the lab by reading our [onboarding](https://github.com/thedonglab/onboarding/blob/master/onboarding.md) document. 

### Postdoctoral Research Fellow

The Dong Lab welcomes applications for both _**computational**_ and _**experimental**_ postdoctoral positions to work on our [projects]({{ site.baseurl }}/projects). In general, we ask you to send your CV, and a cover letter to state your career goal, your research interest, and why you think you are a good fit for the lab, together with the contact information (name, affiliation, email, phone, your relationship) of three references. 
Below are general job descriptions for postdoc positions:

#### Postdoc in Computational Genomics + AI
Do you have a computational background and a strong interest in human genomics? Are you obsessed with how the human genome works in the brain? Do you like to work on big multi-omics data with AI model to find cures or early biomarkers for human brain diseases? If your answer is YES to either question, we have a position for you. In Dong Lab, we'd like to work with you to support your career development by providing opportunities to join large-scale brain disease consortia, access multi-omics datasets from the advanced sequencing technologies, immersive team environment with computational support, and growth plan to independence. We have several projects with well-funded grant support (NIH R01, U19, ASAP etc.) 

**Your tasks**: By joining the bioinformatics team and working together with molecular biologists from our or collaborator's lab, the Postdoctoral Fellow will perform a variety of standard analysis (such as RNA-seq, ATAC-seq, various Quantitative Trait Locus (QTL), alternative splicing and polyadenylation), or advanced integrative analysis such as Transformer-based AI models at a bulk, single-cell, or spatial level.

Other responsibilities include:
-	Develop and implement algorithms and statistical methods to analyze large-scale sequencing data at a bulk, single-cell, or spatial level
-	Design and develop innovative data visualization methods to support genome analysis 
-	Publish and present novel research findings in academic journals and conferences 
-	Develop and maintain collaborations both within Yale and also with outside researchers in academia and industry.

**Your qualifications**: Applicants are expected to have Ph. D. or equivalent doctoral degree in bioinformatics, computer science, genomics, physics, or mathematics. Ideal applicants should have strong programming skills in (Python & R & (Perl \| C \| C++ \| Matlab \| Java)) and good knowledge in algorithms and tools development. Experience in applying LLM to omics is a big plus. Prior experience in analyzing high-throughput sequencing data (RNA-seq, ATAC-seq, small RNA sequencing etc.) is preferred. Advanced usage of large biological repositories, such as UCSC Genome Browser, Ensembl is a plus. Knowledge of gene regulation or AI modeling is a plus. Experience in GWAS, TWAS, eQTL/pQTL/aQTL, allele-specific gene expression, and alternative splicing is a plus. Knowledge or experience in single-cell or spatial omics analysis is a plus. Excellent English communication skills are required. Prior experience in neurology or neurosciences is not required, but helpful. Candidates must have a track record in publication (e.g. at least one first-author publication). Furthermore, the ability to work in a team and experience in the supervision of students is an asset.

#### Postdoc in brain organoid and RNA biology
We have successfully developed midbrain organoids from iPSC derived from healthy controls and neurodegenerative disease patients (manuscript in preparation), and measured the time-series transcriptome of extracellular vesicles (EVs) secreted along with the differentiation. Single-cell RNA-seq, ATAC-seq, and spatial transcriptomics will also be conducted for the generated organoids. Meanwhile, working with our collaborators, we will apply a high-throughput 3D mini-brain platform to capture the organoidâ€™s electrophysiology signals in a real-time manner. Based on our recent publications (e.g., Dong et al., Nature Neuroscience, 2018; Chen et al. Nature Methods, 2021; Dong et al. Nature Communications, 2023), we will use the _ex vivo_ organoid model to test the therapeutic candidates (e.g. drug and RNAs) for Parkinson's and other brain disorders.  

We are also interested in studying EV biology and its implications for early diagnosis and therapeutics of brain diseases. Utilizing the ultra-fast EV isolation system "EXODUS" that we recently developed together with our collaborators (Chen et al. Nature Methods, 2021), we will isolate EVs and profile their total RNAs for over 1000 PD blood samples (funded by NIH R24). It will provide the largest resource of exosome study in the PD community.

**Your task**: By working with collaborators and core facilities, the Postdoctoral Fellow will maintain the organoid lines and perform bulk, single-cell, and spatial omics sequencing experiments for organoids and their culture medium for biomarker and therapeutics purposes.

Other responsibilities include:
-	Design and perform various wet lab experiments (e.g. PCR, ELISA, imaging staining, drug treatment etc.)
-	Assist and work with computational biologists to understand sequencing data analysis result
-	Publish and present novel research findings in academic journals and conferences 
-	Develop and maintain collaborations both within Yale and also with outside researchers in academia and industry.

**Your qualifications**: Applicants are expected to have Ph. D. or equivalent doctoral degree in molecular biology, neuroscience, bioengineering, etc. Ideal applicants should have strong wet lab skills in molecular biology and good knowledge in iPSC or organoid development. Experience in brain organoids is a big plus. Prior experience in performing high-throughput sequencing experiments (RNA-seq, ATAC-seq, small RNA sequencing, etc.) is preferred. Knowledge of gene regulation and RNA biology is a plus. Knowledge or experience in single-cell or spatial omics experiments is a plus. Excellent English communication skills are required. Prior experience in neurology or neurosciences is encouraged. Candidates must have a track record in publication (e.g. at least one first-author publication). Furthermore, the ability to work in a team and experience in the supervision of students is an asset.

### Research Scientist
This Research Scientist faculty position at Yale will be offered a competitive salary and benefits, and expected to provide computational genomics support and service to the lab members and faculty in the local community. As a full-time employee in the well-funded Dong Lab and Adam Center for Parkinson's disease, the candidate will work with the director to develop bioinformatics pipelines for NGS data (e.g., bulk tissue and single-cell RNAseq, ATAC-seq, WGS/WES, ChIP-seq, Hi-C, etc.), provide both standard and customized data analysis service and bioinformatics training to the community. Experience in processing raw FASTQ files is required. Fluent programming in R, Linux bash, and one of the other languages (Python, Perl, C/C++, Matlab, or Java) is required. Experience in workflow/pipeline construction (e.g., Nextflow, Snakemake, CWL/WDL), cloud computing (e.g. AWS, GCP), Docker, or Conda is a plus. Ability to generate high-quality reports, communicate with clients effectively, good time management are key characteristics of an ideal candidate.

### Graduate Students

The Yale University has both an umbrella program and discipline-specific ones.
The umbrella program is the [Biological and Biomedical Sciences (BBS)](https://medicine.yale.edu/bbs/).
Our primary program affiliation is Yale University's [Computational Biology and Biomedical informatics](https://cbb.yale.edu).
We are also excited to welcome students who want to pursue computational research from [other graduate programs](https://www.yale.edu/admissions/graduate-professional-schools) on a case-by-case basis.

Our goal is to help our students develop both a deep familiarity with the computational methods required for data-intensive science and a strong understanding of one or more biological application areas.
We provide training in all aspects through group meetings, individual meetings, and a supportive lab environment.

If you have not yet joined one of the PhD programs above, the first step to joining our group is to apply and be admitted to one of these programs.
If you are already a PhD student in one of the above programs and are interested in rotating in our lab, please [contact Dr Dong]({{ site.baseurl }}/contact) to set up a time to discuss your interest.


<!-- ### Programmers

We work with programmers at [BIDS's bioinformatics engineering team](https://cu-dbmi.github.io/set-website/) to bring research products to practice.
Visit their website to learn about opportunities on the team.
 -->

### Undergraduate Students / Internship / Postgraduate

We welcome undergraduate students into our lab.
Undergraduate researchers have become first authors on papers submitted on research that they performed as part of our group.
We are happy to train undergraduates in many aspects of data-intensive biology, and we have high expectations for their level of commitment to research.

If you are an undergraduate interested in discussing research opportunities please [contact Dr Dong]({{ site.baseurl }}/contact).

{% include figure.html image="images/group-photos/IMG_1547.jpg" width="100%" %}        